pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -164,13 +170,31 @@ func (ic *ifConfigurator) createContainerLink(endpointName string, result *curre
 	if err != nil {
 		return nil, err
 	}
+	containerIPStr, err := parseContainerIPs(result.IPs)
+	if err != nil {
+		klog.Errorf("Failed to find container %s IP", containerID)
+	}
+	ifaceConfig := interfacestore.NewContainerInterface(
+		endpointName,
+		containerID,
+		podName,
+		podNamespace,
+		nil,
+		containerIPStr)
+	ovsAttachInfoData := BuildOVSPortExternalIDs(ifaceConfig)
+	ovsAttachInfo := make(map[string]string)
+	for k, v := range ovsAttachInfoData {
```

Comment:
Could you add some comments to explain when the information is consumed?

Reply:
Done.

Reply:
Do you think we should mention this is for the containerd mainly?

Reply:
Sure. emphasize the ContainerD in comment.